### ISSUE TITLE
[Request] Backport to 1.19.X

### ISSUE THREAD

**Issue by SK1Y101** [blocovermelho/ae2-emi-crafting#22]
I'm currently playing 1.19.2, and am really missing this functionality. Would greatly appreciate a backport if possible.

**Comment by Ftrless** [comment]
Working on it right now, once it's done and minimally tested I'll do pr here

**Comment by Ftrless** [comment]
https://github.com/Ftrless/ae2-emi-crafting/tree/1.19

It seems to be working properly

**Comment by roridev** [comment]
I will pull those changes into a new branch once I get home, just keep that branch alive and do any changes you need to

**Comment by roridev** [comment]
Is Shift+RMB supposed to do anything on EMI?, IIRC only Shift+LMB fills the recipe.

**Comment by Ftrless** [comment]
Oh, yeah, LMB

**Comment by roridev** [comment]
Yeah, there is no easy way to fix that. EMI will happily request ae2 for every single possible recipe on the item list, which causes a lot of lag (#23).

EMI would need to add another context for shift clicking the recipe or make that action trigger the same context as `FILL_RECIPE`.

For now Shift+LMB will be broken until I submit a patch for EMI, which likely will take some time and emily would need to accept that.

**Comment by roridev** [comment]
Fixed
---
Dev: 74d844e
MR: Waiting for release
CF: Waiting for release

**Comment by Slayer5934** [comment]
Looking forward to this
